File: ios/MendixBackwardsCompatUtility/UnsupportedFeatures.m
#include "UnsupportedFeatures.h"

@implementation UnsupportedFeatures
@synthesize reloadInClient;
@synthesize hideSplashScreenInClient;

- (id)init:(BOOL)reloadInClientNotSupported {
    self.reloadInClient = reloadInClientNotSupported;
    return self;
}

- (id)init:(BOOL)reloadInClientNotSupported hideSplashScreenInClient:(BOOL)hideSplashScreenInClientNotSupported {
    self = [self init:reloadInClientNotSupported];
    self.hideSplashScreenInClient = hideSplashScreenInClientNotSupported;
    return self;
}

@end

File: ios/Modules/MendixBackwardsCompatUtility/UnsupportedFeatures.swift
import Foundation

public class UnsupportedFeatures: NSObject {
    public let reloadInClient: Bool
    public let hideSplashScreenInClient: Bool
    
    public init(reloadInClient: Bool, hideSplashScreenInClient: Bool = false) {
        self.reloadInClient = reloadInClient
        self.hideSplashScreenInClient = hideSplashScreenInClient
        super.init()
    }
}
